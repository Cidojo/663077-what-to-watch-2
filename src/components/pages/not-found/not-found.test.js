import * as React from 'react';
import {NotFound} from './not-found.jsx';
import * as renderer from 'react-test-renderer';

it(`should render NotFound component without errors`, () => {
  const tree = renderer.create(<NotFound/>).toJSON();
  expect(tree).toMatchSnapshot();
});
