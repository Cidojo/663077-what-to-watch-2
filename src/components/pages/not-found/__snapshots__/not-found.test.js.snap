// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound component without errors 1`] = `
<h1>
  404 Page Not Found
</h1>
`;
